apache/incubator-zeppelin
# Update default notebook generateName function for create note dialog

### What is this PR for?
The current default name for a notebook in create dialog is random, which is difficult to remember.
Instead of randomly generating names it's useful to have a prefix "Untitled Note " + seq number.
eg: Untitled Note 1, Untitled Note 2 etc.

### What type of PR is it?
Improvement

### How should this be tested?
Create note with default names populated in the field and check if the naming sequence is correct.
When there are no notes with "Untitled Note <num>" the default name should be "Untitled Note 1", otherwise it should be max num + 1.

### Screenshots
Before
<img width="659" alt="before" src="https://cloud.githubusercontent.com/assets/2031306/12294473/c63d823a-ba20-11e5-8e7d-6d9df098ab05.png">
<hr>
<img width="632" alt="after1" src="https://cloud.githubusercontent.com/assets/2031306/12294476/cff636dc-ba20-11e5-82a1-dab7ade0fc7c.png">
<img width="637" alt="after2" src="https://cloud.githubusercontent.com/assets/2031306/12294478/d3c5232c-ba20-11e5-8265-69ce41a1370b.png">
